boofilsic/settings.py

```diff
@@ -231,15 +277,30 @@
 LUMINATI_USERNAME = 'lum-customer-hl_nw4tbv78-zone-static'
 LUMINATI_PASSWORD = 'nsb7te9bw0ney'
 
+SCRAPING_TIMEOUT = 90
+
 # ScraperAPI api key
 SCRAPERAPI_KEY = 'wnb3794v675b8w475h0e8hr7tyge'
+PROXYCRAWL_KEY = None
+SCRAPESTACK_KEY = None
 
 # Spotify credentials
 SPOTIFY_CREDENTIAL = "NzYzNkYTE6MGQ0ODY0NTY2Y2b3n645sdfgAyY2I1ljYjg3Nzc0MjIwODQ0ZWE="
 
 # IMDb API service https://imdb-api.com/
 IMDB_API_KEY = "k23fwewff23"
 
+# The Movie Database (TMDB) API Keys
+TMDB_API3_KEY = "deadbeef"
+TMDB_API4_KEY = "deadbeef.deadbeef.deadbeef"
```

Comment:
Why are there more than one TMDB keys?

Reply:
TMDB has two versions of APIs each using a different key, we are using v3 now, v4 is new but incomplete.

https://developers.themoviedb.org/3/

https://developers.themoviedb.org/4/